flink-dist/src/main/flink-bin/bin/config.sh

```diff
@@ -545,6 +545,34 @@ $FLINK_INHERITED_LOGS
 
 JM_RESOURCE_PARAMS extraction logs:
 jvm_params: $jvm_params
+dynamic_configs: $DYNAMIC_PARAMETERS
+logs: $logging_output
+"
+}
+
+parseTmArgsAndExportLogs() {
+  java_utils_output=$(runBashJavaUtilsCmd GET_TM_RESOURCE_PARAMS "${FLINK_CONF_DIR}" "$FLINK_BIN_DIR/bash-java-utils.jar:$(findFlinkDistJar)" "${ARGS[@]}")
```

Comment:
Can we deduplicate this a bit?
the only differences appear to be `GET_TM_RESOURCE_PARAMS` and `"$@"` vs `"${ARGS[@]}"`

Reply:
Sure.